README.md

```diff
@@ -29,7 +29,6 @@ hatch shell
 Set up your development environment and start the server:
 
 ```
-jlpm setup:dev # only needs to be run once
 jlpm dev
```

Comment:
Isn't this whole section repetitive and incomplete compared to the contributors docs? Should we just remove this and link to the `docs/source/contributors/index.md`?

Reply:
@3coins Good suggestion. I'll follow up with Jason offline about this.